content/en/docs/Writing policies/variables.md

```diff
@@ -324,6 +324,36 @@ Note that certain characters must be escaped for JMESPath processing (ex. `-` in
 
 You can also fetch image properties of all containers for further processing. For example, `{{ images.containers.*.name }}` creates a string list of all image names.
 
+## Inline Variables
+
+Variables may be defined in a `context` for consumption by Kyverno rules. This can be as simple as a static value or an object. The below sets a context variable with a value of `foo`.
+
+```yaml
+    context:
+    # A unique name for the for the variable 
+    # if the user redeclares a variable with the same name it should be re-assigned
+    - name: foodata
+      variable:
+        # value defines the value that the variable must have, it may contain jmespath variables or any yaml object that can be represented as a json object
```

Comment:
All 3 fields (value, default and jmespath) are optional but either `value` or `jmespath` must be defined. `default` can only be used if `jmespath` is defined. All these combinations would automatically be validated during policy creation.

Reply:
Expanded comment to capture this. Resolve if this works.